script.js

```diff
@@ -1,12 +1,104 @@
-//You can edit ALL of the code here
+const rootElem = document.getElementById("root");
+const dropdownEl = document.createElement("select");
+const searchInput = document.createElement("input"); // Creates an input element
+const searchStats = document.createElement("p"); // Counts the number of episodes displayed
+const searchContainer = document.createElement("div");
+const allShows = document.createElement("div"); // A container which displays the episodes
+let totalEpisodes; // Variable for the total number of episodes
+
+// Adds classes to the elements
+searchContainer.classList.add("search-container");
+searchInput.classList.add("search-input");
+searchStats.classList.add("search-stats");
+allShows.classList.add("all-shows");
+
+// Appends the elements
+searchContainer.appendChild(dropdownEl);
+searchContainer.appendChild(searchInput);
+searchContainer.appendChild(searchStats);
+rootElem.appendChild(searchContainer);
+rootElem.appendChild(allShows);
+
+// This event listener goes to the show which is selected in the drop down
+dropdownEl.addEventListener("change", (event) => {
+  document.location = `index.html#${event.target.value}`;
+});
+
+// This function when the page is loaded
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  fetch("https://api.tvmaze.com/shows/82/episodes")
+    .then((response) => response.json())
+    .then((data) => fetchedEpisodes(data))
+    .catch((error) => console.log(error));
+}
```

Comment:
Not sure if we've covered async yet? If not, you can also write this using the async syntax.

```
async function setup() {
  try {
    const response = await fetch("https://api.tvmaze.com/shows/82/episodes")
    const data = await response.json()
    fetchedEpisodes(data)
  } catch (error) {
    console.error(error)
  }
}
```